src/components/ViewDeck/ViewDeck.js

```diff
@@ -1,94 +1,77 @@
-import React, { Component } from "react";
+import React, { Component, useState, useEffect } from "react";
 import { connect } from "react-redux";
 import mapStoreToProps from "../../redux/mapStoreToProps";
 import { PieChart } from "react-minimal-pie-chart";
 import "./ViewDeck.css";
 
-class ViewDeck extends Component {
-  state = {
-    hoverCard: "https://i.stack.imgur.com/Vkq2a.png",
-    updatedQty: "",
-    deckQty: "",
-    parsedCards: [],
-  };
+function ViewDeck(props) {
+  const [state, setState] = React.useState({
+    //STILL NEED STATE
+  });
+
+  const [hoverCard, setHoverCard] = useState(
+    "https://i.stack.imgur.com/Vkq2a.png"
+  );
```

Comment:
If setHoverCard is never used, then this should not be state. It can be a static `const` or even just a static string where needed in the html.